service/controller/v18/resource/ipam/create.go

```diff
@@ -89,51 +112,44 @@ func (r *Resource) allocateSubnet(ctx context.Context) (string, error) {
 
 		awsConfigSubnets, err := getAWSConfigSubnets(r.g8sClient)
 		if err != nil {
-			return "", microerror.Mask(err)
+			return net.IPNet{}, microerror.Mask(err)
 		}
 		reservedSubnets = append(reservedSubnets, awsConfigSubnets...)
 
 		r.logger.LogCtx(ctx, "level", "debug", "message", "found allocated subnets from AWSConfigs")
 	}
 
-	reservedSubnets = canonicalizeSubnets(r.networkRange, reservedSubnets)
+	reservedSubnets = ipam.CanonicalizeSubnets(r.networkRange, reservedSubnets)
```

Comment:
I stumbled upon this and decided to remove copy from here and use `ipam` one now that it's available.